locales/en/messages.json

```diff
@@ -3008,6 +3008,47 @@
     "failsafeProcedureItemSelect2": {
         "message": "Drop"
     },
+    "failsafeProcedureItemSelect4": {
+        "message": "GPS Rescue"
+    },
+
+    "failsafeGpsRescueItemAngle": {
+        "message": "Angle"
+    },
+    "failsafeGpsRescueItemInitialAltitude": {
+        "message": "Initial altitude (meters)"
+    },
+    "failsafeGpsRescueItemDescentDistance": {
+        "message": "Descent distance (meters)"
+    },
+    "failsafeGpsRescueItemGroundSpeed": {
+        "message": "Ground speed"
```

Comment:
I think we should add the unit here as well.

Reply:
True, looking at the wiki, is centimeters per second, I will change it and update the PR.

Reply:
Thinking it again, maybe is better to transform it to m/s, is a unit more habitual for speed and maybe is better understood by the user.